<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current register write list</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QPlainTextEdit" name="registers_field"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="setb">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearb">
            <property name="text">
             <string>Clear all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="doneb">
            <property name="text">
             <string>Done</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="manual_edit">
            <property name="text">
             <string>Manual Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,1">
        <item>
         <widget class="QLineEdit" name="address">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="data">
          <property name="placeholderText">
           <string>Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mask">
          <property name="placeholderText">
           <string>Mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>124</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>124</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
